.github/workflows/ci.yml

```
@@ -94,7 +94,7 @@ jobs:
         cat /proc/cpuinfo
         ./util/opensslwrap.sh version -c
     - name: make test
-      run: .github/workflows/make-test
+      run: .github/workflows/make-test || touch .test_failure
```

Please use this instead:
```suggestion
      run: .github/workflows/make-test
      continue-on-error: true
```